README.md

```diff
@@ -37,14 +37,25 @@ It’s a Rails app that:
 
     If you’d like to do the setup manually, see [advanced setup](#advanced-setup) below.
 
-8. Finally, start the server!
+8. Start the server!
 
     ```sh
     $ bundle exec rails server
     ```
 
     You should now have a server running and can visit it at http://localhost:3000/. Open that up in a browser and go to town!
 
+9. Bulk importing (and, in the future, potentially other features) make use of a
+   Redis queue. If you plan to use any of these features, you must also start a
+   Redis server and worker.
```

Comment:
Might be helpful to include basic install instructions (e.g. `brew install redis` on OS X, `apt-get install redis` on Debians) and/or a link to https://redis.io for people who don’t know Redis.